packages/waku/package.json

```diff
@@ -97,6 +97,9 @@
   "devDependencies": {
     "@netlify/functions": "^3.0.0",
     "@swc/cli": "^0.5.2",
+    "react": "19.0.0",
+    "react-dom": "19.0.0",
+    "react-server-dom-webpack": "19.0.0",
```

Comment:
Can't we remove this?

Reply:
this is necessary as a devDep. `pnpm update` does not apply to peerDeps, so without this the build ends up with two versions of react: react canary & whatever resolves to `~19.0.0` at time of the test run.

Reply:
got it.